app/views/gradebooks/speed_grader.html.erb

```diff
@@ -158,11 +158,13 @@
             <div id="submission_files_list">
               <div id="submission_file_hidden"  class="submission-file" style='display:none;'>
                 <span class="turnitin_score_container"></span>
+                <span class="vericite_score_container"></span>
```

Comment:
Since turnitin and vericite are mutually exclusive, let's use the same container for both.